example/windows/agent_config_promtail.yaml

```diff
@@ -0,0 +1,22 @@
+server:
```

Comment:
I'd personally delete this file and inline it into docs/windows.md, like in my suggested edit.